install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -12,8 +12,8 @@ rollingMaxUnavailable: 25%
 image: pilot
 sidecar: true
 traceSampling: 1.0
-# if protocol sniffing is enabled. Default to false.
-enableProtocolSniffing: false
+# if protocol sniffing is enabled. Default to true.
```

Comment:
I don't follow why you need a variable if there is nothing to configure?

Reply:
What do you mean nothing to configure? This is used on L112 `            value: "{{ .Values.enableProtocolSniffing }}"`